schema/partner.js

```diff
@@ -86,6 +91,24 @@ const PartnerType = new GraphQLObjectType({
           }));
         },
       },
+      artworks: {
+        type: new GraphQLList(Artwork.type),
+        args: {
+          size: {
+            type: GraphQLInt,
+          },
+          sort: ArtworkSorts,
+          exclude: {
```

Comment:
FYI the ES endpoint supports excluding artwork id's, but the param isn't `exclude`, its `exclude_artwork_ids`.

https://github.com/artsy/gravity/blob/7492ff7b2f406eb35745541502640c85b56042a4/app/api/v1/filter_endpoint.rb#L20

You won't need to manually exclude them from the response.